features/basic-login.feature

```diff
@@ -6,7 +6,8 @@ Feature: Basic end-to-end test
   Scenario: Raising an exception message
```

Comment:
Scenario name is a bit confusing, assuming from the file name that we testing the login here?

Reply:
Yeah, I got the impression from the PNC errors check that this is also testing if the case details page loads okay?

Reply:
ah wait I see what you mean about the scenario name now, will fix that